handler.go

```diff
@@ -690,10 +689,50 @@ func (fm *Frontman) processInput(input *Input, resultsChan chan<- Result) {
 		}(check)
 	}
 
+	for _, check := range input.SNMPChecks {
+		wg.Add(1)
+		go func(check SNMPCheck) {
+			defer wg.Done()
+
+			if check.UUID == "" {
```

Comment:
Is it worth to use some external package to wrap UUID? It could validate for example it's actually UUID not just a string

Reply:
The property called "UUID" here is just a string, see "checkUUID" in https://github.com/cloudradar-monitoring/frontman/blob/master/example.json#L3

Reply:
Also, this whole function was based on existing functions, see 

https://github.com/cloudradar-monitoring/frontman/blob/master/handler.go#L620

and

https://github.com/cloudradar-monitoring/frontman/blob/master/handler.go#L656